src/server.h

```diff
@@ -2105,6 +2105,7 @@ void ACLClearCommandID(void);
 user *ACLGetUserByName(const char *name, size_t namelen);
 int ACLCheckCommandPerm(client *c, int *keyidxptr);
 int ACLCheckPubsubPerm(client *c, int idx, int count, int literal, int *idxptr);
+int ACLCheckAllPerm(client *c, int *idxptr);
```

Comment:
Yeah, please delete the other two lines, we should just expose this one if we don't want anyone calling the above two now.